src/Areas/BicepSchema/Services/ResourceProperties/Entities/UniqueResourceType.cs

```diff
@@ -5,6 +5,7 @@
 using AzureMcp.Areas.BicepSchema.Services.ResourceProperties.Helpers;
 
 namespace AzureMcp.Areas.BicepSchema.Services.ResourceProperties.Entities;
+
```

Comment:
Minor note: I'm not sure if format changes to other files are needed in this PR. It might make sense to handle those separately, so we can keep this PR focused on NuGet-related updates.